core/save-profiledata.c

```diff
@@ -148,13 +158,45 @@ static void put_headers(struct membuffer *b)
 	put_csv_string(b, "icd_warning");
 }
 
+static void put_st_event(struct membuffer *b, struct plot_data *entry, int offset, int length)
+{
+	double value;
+	int decimals;
+	const char *unit;
+
+	if (entry->sec < offset || entry->sec > offset + length)
+		return;
+
+	put_format(b, "Dialogue: 0,");
+	put_secs(b, entry->sec - offset);
+	put_secs(b, (entry+1)->sec - offset < length ? (entry+1)->sec - offset : length);
+	put_format(b, "Default,,0,0,0,,");
+	put_format(b, "%d:%02d ", FRACTION(entry->sec, 60));
+	value = get_depth_units(entry->depth, &decimals, &unit);
+	put_format(b, "D=%02.2f %s ", value, unit);
+	if (entry->temperature) {
+		value = get_temp_units(entry->temperature, &unit);
+		put_format(b, "T=%.1f%s ", value, unit);
+	}
+	if (entry->ndl < 3600) {
```

Comment:
interesting choice... I'd say that deserves a comment?

Reply:
The comment is in the commit message: this is a todo. I wrote this on a train and wanted to get it out before the train arrives.

Reply:
I understand the desire to get a PR out. But I don't see an explanation for the choice between NDL and TTS in the commit message, either...